src/index.ts

```diff
@@ -141,11 +145,25 @@ async function run() {
   }
 }
 
-log("Starting service");
 
-cron.schedule(config.cron, run);
+function init() {
+  console.log('init - why not use pino')
```

Comment:
Why add a pino where the only thing that would be useful from it is time. Use the log function.